Algorithmia/algorithm.py

```diff
@@ -29,6 +37,97 @@ def set_options(self, timeout=300, stdout=False, output=OutputType.default, **qu
         self.query_parameters.update(query_parameters)
         return self
 
+    # Create a new algorithm
+    def create(self, details={}, settings={}, version_info={}):
+        detailsObj = Details(**details)
+        settingsObj = SettingsMandatory(**settings)
+        createRequestVersionInfoObj = CreateRequestVersionInfo(**version_info)
+        create_parameters = {"name": self.algoname, "details": detailsObj, "settings": settingsObj, "version_info": createRequestVersionInfoObj}
+        create_request = CreateRequest(**create_parameters)
+        try:
+            # Create Algorithm
+            api_response = self.client.manageApi.create_algorithm(self.username, create_request)
+            return api_response
+        except ApiException as e:
+            error_message = json.loads(e.body)["error"]["message"]
+            raise ApiError(error_message)
+
+    # Update the settings in an algorithm
+    def update(self, details={}, settings={}, version_info={}):
+        detailsObj = Details(**details)
+        settingsObj = Settings(**settings)
+        createRequestVersionInfoObj = CreateRequestVersionInfo(**version_info)
+        update_parameters = {"details": detailsObj, "settings": settingsObj, "version_info": createRequestVersionInfoObj}
+        update_request = UpdateRequest(**update_parameters)
```

Comment:
Small nit:
Some of your variables are camel case, and some are snake case - I think most of the time you snake case for dicts, but not always. Might be useful to be consistent here.

I think the python default is always snake case?

Reply:
Good point. Also as far as I can tell, the client has used both :/

Look at line 19-30. I could change the style for the code I wrote. I personally prefer using underscores. It makes reading the code easier.

Reply:
Per python style guide variables should be snake case:

https://www.python.org/dev/peps/pep-0008/#function-and-variable-names